src/test/mod.rs

```diff
@@ -202,14 +204,20 @@ fn init_env(project_dir: &str) {
     env::set_current_dir(cwd).expect(FAIL_MSG);
 }
 
+#[derive(Clone, Debug, PartialEq)]
+enum Id {
```

Comment:
Do you want to reuse the Id struct from the main code, or should there be a separation like here?